src/apps/code-editor/src/app/components/FileActions/components/Delete/Delete.js

```diff
@@ -24,11 +24,19 @@ export const Delete = memo(function Delete(props) {
     <div className={styles.DeleteBtn}>
       {props.fileName !== "loader" ? (
         <Button
-          type="warn"
+          variant="contained"
+          color="error"
           onClick={() => setOpen(true)}
-          className={styles.Button}
+          startIcon={<DeleteIcon />}
+          sx={{
```

Comment:
Should be sticking to theme and not defining colors. You can use them like `palette.primary`